npc/re/guild/mission_main.txt

```diff
@@ -0,0 +1,434 @@
+								else
+									mes "Currently, the total number of fortress that your guild has conquered";
+								for(.@i = 0; .@i < 5; ++.@i){
+									mes getcastlename(.castle$[.@i+.@zoneid*5]) + (.@gid == getcastledata(.castle$[.@i+.@zoneid*5],0) ? ": ^4d4dffOccupied^000000" : "");
+								}
+								if(.@total == 5)
+									mes "Your guild has conquered " + .region$[.@zoneid] + "!";
+								else
+									mes "is " + .@total + "!";
+								next;
+								mes .@npcname$;
+								mes "I'll give you a reward accordingly. I hope you make good use of it.";
+								mes "You have an hour of waiting time until the next report. Keep that in mind and try again.";
+								setquest .conquer_delay_questid[.@zoneid];
+								getitem 6489,.@total;
+								getitem 6615,.@total;
+								getitem 12888,.@total;
+								close;
+							}
+						}
+					} else {
+						mes .@npcname$;
+						mes "I'm sorry but you can complete this mission only during the siege hours. I'm afraid I can't do anything here.";
+						close;
+					}
+				}
+			} else {
+				mes .@npcname$;
+				mes "I'm sorry but conquering missions are for guild masters only.";
+				mes "Members should try guild dungeon daily quests instead.";
+				close;
+			}
+			break;
+		case 2: // Daily Quests
+			mes .@npcname$;
+			mes "These are missions that you can complete in guild dungeons of " + .region$[.@zoneid] + " area.";
+			mes "You can cancel them if you want to. Feel free to challenge yourself.";
+			next;
+			.@cancel_index = (.@edition == 1 ? 3 : 2);
+			for(.@i = 0; .@i < 3; ++.@i){
+				if(.daily_hunting_questid[.@zoneid*3 + .@i])
+					.@menu$ = .@menu$ + .region$[.@zoneid] + " Mission " + (.@i+1) + (isbegin_quest(.daily_hunting_questid[.@zoneid*3 + .@i]) ? " ^4d4dffCheck the result^000000" : "") + ":";
+			}
+			.@sel = select(.@menu$) - 1;
+			.@index = .@zoneid*3 + .@sel;
+			if(.@sel == .@cancel_index){
+				mes .@npcname$;
+				mes "You don't need to rush since these missions take place in guild dungeons.";
+				mes "Feel free to give it a try whenever you want to.";
+				close;
+			}
+			if(BaseLevel < .level_req[.@index]){
+				mes .@npcname$;
+				mes "Mm, unfortunately, this mission is limited to users with ^4d4dff level " + .level_req[.@index] + " ^000000 or higher only.";
+				mes "Your level is not high enough.";
+				close;
+			}
+			.@hunting = checkquest(.daily_hunting_questid[.@index], HUNTING);
+			.@playtime = checkquest(.daily_delay_questid[.@index], PLAYTIME);
+			if(.@hunting > -1){
+				if(.@hunting == 2){
+					mes .@npcname$;
+					mes "Oh, you've done a good job.";
+					mes "It isn't much, but hopefully it will be helpful.";
+					setquest .daily_delay_questid[.@index];
+					erasequest .daily_hunting_questid[.@index];
+					.@hasFixedBonus = (.@zoneid > 3);
+					.@TErestricted = (.@zoneid == 4 || .@zoneid == 5);
+					getexp calcexp(.max_level[.@index],(.@zoneid<4 && !((.@index+1)%3))),calcjobexp(.@hasFixedBonus,.@TErestricted);
```

Comment:
Judging from your parameters:
Wouldnt it be nicer to hand over the index or zoneid and do those checks there?